client/app/scripts/liveblog-bloglist/module.js

```diff
@@ -12,6 +12,7 @@
         ];
         $scope.activeState = isArchivedFilterSelected ? $scope.states[1] : $scope.states[0];
         $scope.creationStep = 'Details';
+        $scope.requestAccessMessage = gettext('Click to request access');
```

Comment:
I am using it as a tooltip for a <li> element and as part of an expression something like title="{{ !isUserAllowedToOpenBlog(blog) ? requestAccessMessage: '' }}" so it's kindof hard to translate directly in html